openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -61,7 +65,12 @@ function download_ingress {
     index=$(( i+1 ))
     file="${files[$i]}.yaml"
     ingress_target_file="$ingress_dir/$index-$file"
-    url="https://raw.githubusercontent.com/knative-sandbox/${component}/${version}/config/${file}"
+
+    if [[ "${component}" == "net-istio" ]]; then
```

Comment:
is somethign else using this? If not, let's drop the `if`

Reply:
Yeah I dont think so.